rplugin/python3/deoplete/child.py

```diff
@@ -484,6 +484,8 @@ def _set_source_attributes(self, context):
             'matchers',
             'max_abbr_width',
             'max_candidates',
+            'max_abbr_width',
```

Comment:
`max_abbr_width` is not needed.  Because, it is already set.
Please see the code.